pallets/parachain-staking/src/tests/test_growth.rs

```diff
@@ -540,6 +540,7 @@ mod growth_amount {
                     let amount = 400;
                     assert_ok!(ParachainStaking::payout_collators(amount, PERIOD_INDEX));
 
+                    assert_ok!(ParachainStaking::payout_collators(amount, PERIOD_INDEX));
```

Comment:
Is this test supposed to be calling the payout twice like this?

Reply:
No, it shouldn't. Removed.